src/main.c

```diff
@@ -277,6 +311,7 @@ static handler_fn_t* lookupHandler(uint8_t ins) {
 // and sent in the next io_exchange call.
 static void sia_main(void) {
 	// Mark the transaction context as uninitialized.
+	explicit_bzero(&global, sizeof(global));
```

Comment:
IIRC the C standard requires that these variables be zero-initialized, but whatev